content/governance.md

```diff
@@ -34,19 +34,16 @@ New members may be proposed by any existing member by email to [prometheus-team]
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to [prometheus-team][team] for record-keeping purposes.
 
-If they choose to accept, the following steps are taken:
-
-* Team members are added to the [GitHub organization][gh] as _Owner_.
-* Team members are added to the [team mailing list][team].
-* Team members are added to the list of team members in this document.
-* New team members are announced on the [developers mailing list][devs] by an existing team member.
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
 Team members may retire at any time by emailing [the team][team].
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote, does not count towards the quorum, and may not make any changes to this and related documents any more.
```

Comment:
Perhaps clarify that the procedure is separate for each individual member? I vaguely remember the joke at the dev summit that went like: "I want to vote everybody out except me. I'm the only one eligible to vote, and I vote for it."

Reply:
It seems like changing the document is the least damage an harmful member could do. Maybe it would be better in case of a force removal vote, that the team member gets its owner rights dropped in github and the requester also drops his owner rights at the same time. Or just leave it as it.

Reply:
Boilerplated.